demos/interactive_face_detection_demo/utils.hpp

```diff
@@ -0,0 +1,125 @@
+// Copyright (C) 2018-2019 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+# pragma once
+
+#include <gflags/gflags.h>
+#include <functional>
+#include <iostream>
+#include <fstream>
+#include <random>
+#include <memory>
+#include <chrono>
+#include <vector>
+#include <string>
+#include <utility>
+#include <algorithm>
+#include <iterator>
+#include <map>
```

Comment:
Please put the 80% of these includes to the appropriate .cpp file. I am not sure if you need all them in the header.

Reply:
Removed all unnecessary includes

Reply:
Ok